cypress/e2e/UserStoryTemplate.cy.js

```diff
@@ -0,0 +1,28 @@
+// UserStory Test
+describe("See profile Test", () => {
+  // Perform Login before EACH UserStory test because there could be things 
+  // that happen in a previous test that affects the results of the current 
+  // test. 
+  // With a fresh login, a clean state is ensured every time. 
+  before(() => {
+    cy.visit(Cypress.env("REACT_APP_GRAASP_COMPOSE_HOST_STAGE"));
+    cy.url().should(
+      "be.equal",
+      Cypress.env("REACT_APP_AUTHENTICATION_HOST_STAGE") + "/signin"
+    );
+    cy.login(Cypress.env('TEST_EMAIL'), Cypress.env('TEST_PASSWORD')); // custom command that hanldes signIn wth password
```

Comment:
I think you can include this whole block in a command.

Reply:
I moved the url check to the signIn command. I think we should leave the visit command here, as in this user story we are visiting COMPOSE, but in others we may want to visit other hosts